tools/proxysql_galera_checker.sh

```diff
@@ -1,41 +1,99 @@
 #!/bin/bash
-
 ## inspired by Percona clustercheck.sh
-if [ "$1" = '-h' -o "$1" = '--help'  -o -z "$1" ]; then
-  echo "Usage: $0 <hostname> <port> <hostgroup_id> <log_file>" && exit 0
+
+# CHANGE THOSE
+PROXYSQL_USERNAME="admin"
+PROXYSQL_PASSWORD="admin"
+PROXYSQL_HOSTNAME="localhost"
+PROXYSQL_PORT="6032"
+#
+
+if [ "$1" = '-h' -o "$1" = '--help'  -o -z "$1" ]
+then
+  usage
 fi
-if [ $# -lt 3 ]; then
+
+if [ $# -lt 1 ]
+then
   echo "Invalid number of arguments"
-  echo "Usage: $0 <hostname> <port> <hostgroup_id> <log_file>" && exit 1
+  usage
```

Comment:
Likewise, usage function is has not yet been defined. It has to be moved before line 13th